docs/content/concepts/ops-jobs-graphs/jobs.mdx

```diff
@@ -0,0 +1,262 @@
+---
+title: Jobs | Dagster
+description: A job is an executable graph of ops, with optional resources and configuration.
+---
+
+# Jobs
+
+Jobs are the main unit of execution and monitoring in Dagster. The core of a job is a [graph](/concepts/ops-jobs-graphs/graphs) of [ops](/concepts/ops-jobs-graphs/ops) connected via data dependencies.
+
+Ops are linked together by defining the dependencies between their inputs and outputs. An important difference between Dagster and other workflow systems is that, in Dagster, op dependencies are expressed as data dependencies, not just execution dependencies.
+
+This difference enables Dagster to support richer modeling of dependencies. Instead of merely ensuring that the order of execution is correct, dependencies in Dagster provide a variety of compile and run-time checks.
+
+Using jobs, you can:
+
+- Materialize a selection of [assets](/concepts/assets/software-defined-assets)
+- Determine op behavior using [resources](/concepts/resources) and [configuration](/concepts/configuration/config-schema)
+- Define [schedules](/concepts/partitions-schedules-sensors/schedules) to execute jobs at fixed intervals
+- Define [sensors](/concepts/partitions-schedules-sensors/sensors) to trigger jobs when external changes occur
+
+---
+
+## Relevant APIs
+
+| Name                                | Description                                                                                                                                                     |
+| ----------------------------------- | --------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| <PyObject object="job" decorator /> | The decorator used to create a job.                                                                                                                             |
+| <PyObject object="JobDefinition" /> | A job definition. Jobs are the main unit of execution and monitoring in Dagster. Typically constructed using the <PyObject object="job" decorator /> decorator. |
+|                                     |                                                                                                                                                                 |
+
+---
+
+## Creating jobs
+
+Jobs can be created in several ways:
+
+- [Using the <PyObject object="job" decorator /> decorator](#using-the-job-decorator)
+- [From assets](#from-assets)
+- [From a graph](#from-a-graph)
+
+### Using the @job decorator
+
+The simplest way to create a job is to use the <PyObject object="job"/> decorator.
+
+Within the decorated function body, you can use function calls to indicate the dependency structure between the ops/graphs. In this example, the `add_one` op depends on the `return_five` op's output. Because this data dependency exists, the `add_one` op executes after `return_five` runs successfully and emits the required output.
```

Comment:
I think it's worth mentioning that, unlike for asset-based jobs, whose topology depends on the assets in the job, you explicitly define the topology of a `@job`-based job (i.e. the dependencies between the ops) when you define the job.